pkg/kubelet/server.go

```diff
@@ -109,6 +109,42 @@ func (s *Server) error(w http.ResponseWriter, err error) {
 	http.Error(w, fmt.Sprintf("Internal Error: %v", err), http.StatusInternalServerError)
 }
 
+func isValidDockerVersion(ver []uint) (bool, string) {
+	minAllowedVersion := []uint{1, 3, 0}
```

Comment:
I don't think that this is what you want now that you are using api version, not server version.  I think that you want []{1, 15} ?